cmd/agent/subcommands/run/dependent_services_windows.go

```diff
@@ -56,10 +56,19 @@ var subservices = []Servicedef{
 			"network_config.enabled":          config.SystemProbe,
 			"system_probe_config.enabled":     config.SystemProbe,
 			"windows_crash_detection.enabled": config.SystemProbe,
+			"runtime_security_config.enabled": config.SystemProbe,
 		},
 		serviceName: "datadog-system-probe",
 		serviceInit: sysprobeInit,
 	},
+	{
+		name: "cws",
+		configKeys: map[string]config.Config{
+			"runtime_security_config.enabled": config.SystemProbe,
```

Comment:
Shouldn't this be `config.Datadog` ? AFAIK the security agent does not read `system-probe.yaml`

Reply:
this is how the core agent decides whether to start the security agent service.  it's not clear that the agent reads teh security agent config.  So this seemed sufficient; if it's configured in system probe, then we should start the security agent service.

Reply:
Ah yes ok it makes more sense this way. Thanks !